### PR TITLE
v0.5.1

### PR DESCRIPTION
**Pull request by khoroshevskyi**
- Fixed update project errors

### CODE REVIEW

File: docs/changelog.md
Comment (ayobi) [inline docs/changelog.md:1]: violation? should be validation right?

File: docs/changelog.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) format.
 
+## [0.5.1] -- 2023-07-10
+- Fixed errors in updating projects, that caused Unique Violation error
```
Comment (ayobi) [inline docs/changelog.md:6]: violation? should be validation right?

File: pepdbagent/modules/project.py [hunk 2]
```diff
@@ -471,7 +472,7 @@ def __create_update_dict(update_values: UpdateItems) -> dict:
             if update_values.description is not None:
                 update_values.config["description"] = update_values.description
             if update_values.name is not None:
-                update_values.config["name"] = update_values.description
+                update_values.config["name"] = update_values.name
```
Comment (ayobi) [inline pepdbagent/modules/project.py:475]: nice catch ;)

Comment (khoroshevskyi) [reply pepdbagent/modules/project.py:475]: Thanks @ayobi for finding it:D

File: docs/changelog.md (hunk 1 above)
Comment (khoroshevskyi) [inline docs/changelog.md:6]:
```suggestion
- Fixed errors in updating projects, that caused Unique Constraint violation error
```

**Review by ayobi — CHANGES_REQUESTED** [review]
lgtm

**Review by ayobi — APPROVED** [review]
lgtm

**Comment by ayobi** [comment]
> lgtm

whoops, meant to approve

**Pull request merged**